### PR TITLE
chore(infra): Q4 2020 update

### PR DESCRIPTION
**Pull request by aabouzaid**
Related to: INFRA-1888

### CODE REVIEW

File: infra.csv [hunk 1]
```diff
@@ -46,6 +52,6 @@ shfmt,Tools,Adopt,TRUE,
 Shellcheck,Tools,Adopt,TRUE,
 Kustomize,Tools,Adopt,FALSE,"<p><a href=""https://github.com/kubernetes-sigs/kustomize"">Kustomize</a> is a YAML templating tool to render k8s resources. Used in <a href=""https://github.com/camunda/infra-core"">infra-core</a> project.</p>"
 config-lint,Tools,Adopt,FALSE,"<p><a href=""https://github.com/stelligent/config-lint"">config-lint</a> is a tool to validate configuration files using rules specified in YAML. We're use it locally and in CI pipelines to lint and ensure Terraform best practices like <a href=""https://www.terraform.io/docs/extend/best-practices/naming.html"">naming convention</a>.</p>"
-Atlantis,Tools,Assess,TRUE,
```
Comment (maxdanilov) [inline infra.csv:49]: maybe move it to `HOLD` instead (with the reasoning)?

File: infra.csv [hunk 2]
```diff
@@ -46,6 +52,6 @@ shfmt,Tools,Adopt,TRUE,
...
-terradiff,Tools,Assess,TRUE,"<p>Can alert when Terraform code does not match state: <a href=""https://github.com/jml/terradiff"">terradiff</a></p>"
```
Comment (maxdanilov) [inline infra.csv:50]: same for terradiff

File: infra.csv [hunk 3]
```diff
@@ -38,6 +43,7 @@ Kubernetes,Languages & Frameworks,Adopt,FALSE,"<p>We're using k8s to host a wide
 Terraform,Languages & Frameworks,Adopt,FALSE,"<p>Used to define infrastructure as code, example: <a href=""https://github.com/camunda/infra-core"">infra-core</a> project.</p>"
 Javascript,Languages & Frameworks,Adopt,TRUE,"<p>Used by the <a href=""https://downloads.camunda.cloud"">Download center</a> for Camunda products. <a href=""https://github.com/camunda-internal/camunda-downloads"">repository</a></p>"
 Node.js,Languages & Frameworks,Adopt,TRUE,"<p>Used by the <a href=""https://downloads.camunda.cloud"">Download center</a> for Camunda products. <a href=""https://github.com/camunda-internal/camunda-downloads"">repository</a></p>"
+
```
Comment (maxdanilov) [inline infra.csv]: does the newline not break things for the visualizer?

File: infra.csv (hunk 1 above)
Comment (aabouzaid) [reply infra.csv:49]: @maxdanilov but we never used it afaik ... it was under evaluation but never used.

File: infra.csv (hunk 3 above)
Comment (aabouzaid) [reply infra.csv]: Looks like it breaks it, so removed them :+1:

**Review by maxdanilov — APPROVED** [review]
LGTM, thank you 👍

**Pull request merged**